github_connector/models/abstract_github_model.py

```diff
@@ -0,0 +1,275 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2016-Today: Odoo Community Association (OCA)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+import urllib
+import logging
+
+from openerp import tools, api, fields, models, exceptions, _
+from .github import Github
```

Comment:
isnot Github an external library that should be checked for existence? (if not forget my comment)